utilities/copyoffload_migration.py

```
@@ -691,3 +738,329 @@ def verify_xcopy_used_per_datastore(
+            labels: dict[str, str] = pod.instance.metadata.labels or {}
+            pvc_name: str = labels.get("pvcName", pod.name)
+            field_selector = f"involvedObject.name={pvc_name},involvedObject.kind=PersistentVolumeClaim"
+            for event in Event.list(
+                client=ocp_admin_client,
+                namespace=target_namespace,
+                field_selector=field_selector,
+                since_seconds=py_config["plan_wait_timeout"],
+            ):
+                if event.get("reason") == POPULATOR_THROTTLED_EVENT_REASON:
+                    seen.add(pvc_name)
+                    LOGGER.info(f"PVC '{pvc_name}' has {POPULATOR_THROTTLED_EVENT_REASON} event")
```

Ruff (0.15.17)

[warning] Logging statement uses f-string

(G004)